.gitignore

```diff
@@ -1,9 +1,5 @@
 /target
 /result
-cfg
-ln1
-ln2
-it
 .idea
```

Comment:
We might also think about removing setup-specific entries and document `.git/info/exclude` somewhere